Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -80,6 +80,16 @@ RGroupDecomposition::RGroupDecomposition(
     const RGroupDecompositionParameters &params)
     : data(new RGroupDecompData(cores, params)) {}
 
+RGroupDecomposition::RGroupDecomposition(const ROMol &inputCore, const std::string &json) {
```

Comment:
I would name the second parameter 'params' instead of 'json'

Reply:
It is stadrad way